agentassist-widget-sdk/UI/agentindex.js

```diff
@@ -6446,29 +6492,43 @@ window.AgentAssist = function AgentAssist(containerId, _conversationId, _botId,
                 function getAutoSearchApiResult(e){
                     console.log(arguments[1],"this","get in auto search api", e)
                     let payload = {
-                        "query": e.target.value,
+                        "query": e.target?.value,
                         "maxNumOfResults": 3,
                         "lang": "en"
                     }
                     let isLibraryTab = arguments[1];
                     $('#overLayAutoSearchDiv').addClass('hide').removeAttr('style');
                     $('#overLayAutoSearch').find('.search-results-text')?.remove();
                     $('.search-block').find('.search-results-text-in-lib')?.remove();
-                    if (e.target.value.length > 0) {                       
+                    let headersVal = {};
+                    if(connectionDetails.isSAT) {
+                        headersVal = {
+                            'Authorization': 'bearer' + ' ' + connectionDetails.tokenVal,
+                            'accountId': connectionDetails.accountId
+                        }
+                    } else {
+                        headersVal = {
+                            'Authorization': result.authorization.token_type + ' ' + result.authorization.accessToken,
+                            "AccountId": result.userInfo.accountId
```

Comment:
@PoojaPr-Kore eAD flag is missing in the if condition, is it optional. in else cond is accountId is needed to send when it is an external third-party AgentDesktops. can you get confirmation from Ketan regarding this? whether it will affect it or not